packages/devextreme/testing/tests/DevExpress.ui.widgets/treeView.scrolling.tests.js

```diff
@@ -226,6 +229,30 @@ QUnit.module('useNativeScrolling', () => {
         assert.equal($treeView.find(`.${SCROLLABLE_NATIVE_CLASS}`).length, 1, 'native scrollable');
         assert.equal($treeView.find(`.${SCROLLABLE_SIMULATED_CLASS}`).length, 0, 'simulated scrollable');
     });
+
+    QUnit.test('tabindex for first node container', function(assert) {
+        const wrapper = new TreeViewTestWrapper({
+            items: [
+                { id: 'item1' },
+                { id: 'item2' },
+                { id: 'item3' },
+            ],
+            useNativeScrolling: true,
+        });
+        const $treeView = wrapper.getElement();
+
+        let $nodeContainer = $treeView.find(`.${NODE_CONTAINER_CLASS}`).first();
+
+        const isIOS = devices.real().ios;
+
+        assert.strictEqual($nodeContainer.attr('tabindex'), isIOS ? '0' : undefined);
+
+        wrapper.instance.option({ height: 100 });
+
+        $nodeContainer = $treeView.find(`.${NODE_CONTAINER_CLASS}`).first();
+
+        assert.strictEqual($nodeContainer.attr('tabindex'), isIOS ? '0' : undefined);
```

Comment:
I don't see any IOS specificity affecting this task.

Reply:
These tests was removed.